packages/element/src/renderElement.ts

```diff
@@ -659,6 +662,16 @@ const drawElementFromCanvas = (
     // revert afterwards we don't have account for it during drawing
     context.translate(-cx, -cy);
 
+    const isDarkMode = appState.theme === 'dark';
+    let isEmojiImage = false;
+if (element.type === "text" && typeof element.text === "string") {
+  isEmojiImage = isEmojiOnly(element.text);
+}
+if (isDarkMode &&  isEmojiImage) {
+  context.filter = 'invert(1) hue-rotate(180deg) saturate(1.2) brightness(1.05)';
```

Comment:
Why try to approximate it instead of applying the direct reverse of THEME_FILTER?